src/player_display.cpp

```diff
@@ -682,7 +682,7 @@ Strength - 4;    Dexterity - 4;    Intelligence - 4;    Perception - 4" ) );
     }
     if( get_hunger() + get_starvation() > 100 ) {
         pen = abs( hunger_speed_penalty( get_hunger() + get_starvation() ) );
-        mvwprintz( w_speed, line, 1, c_red, _( "Inanition           -%s%d%%" ),
+        mvwprintz( w_speed, line, 1, c_red, _( "Malnutrition           -%s%d%%" ),
```

Comment:
Same length
```suggestion
        mvwprintz( w_speed, line, 1, c_red, _( "Malnutrition        -%s%d%%" ),
```

Reply:
Im changing the term again back to its previous value of Hunger before the PR that changed it last time. I also  made sure the length was correct and if not ill fix it here.